ext/mbstring/mbstring.c

```diff
@@ -420,7 +420,7 @@ php_mb_parse_encoding_list(const char *value, size_t value_length, const mbfl_en
 					*entry++ = encoding;
 					n++;
 				} else {
-					php_error_docref(NULL, E_WARNING, "Unknown encoding \"%s\"", p1);
+					zend_argument_value_error(arg_num, "must be a list of valid encoding, \"%s\" given", p1);
```

Comment:
```suggestion
					zend_argument_value_error(arg_num, "must be a list of valid encodings, \"%s\" given", p1);
```

Though I think it is a bit weird, because `%s` is only one of the encodings in the list. Another possibility would be:

```suggestion
					zend_argument_value_error(arg_num, "contains invalid encoding \"%s\"", p1);
```